uvicorn/supervisors/basereload.py

```diff
@@ -36,6 +43,7 @@ def run(self):
                 self.restart()
         self.shutdown()
 
+
```

Comment:
Last nit 🙏  (don't know if this would make CI fail, probably not as `flake8` isn't here yet)

Reply:
it was failing black, I should have tested it ....